### DISCUSSION TITLE
Add SwinIR v2 model in the Stable Diffusion Web UI (`auto`)

### DISCUSSION THREAD

**Discussion by mrauhu** [AbdBarho/stable-diffusion-webui-docker#222] [Ideas]
Hello, @AbdBarho.

Can you add the [Swin2SR_RealworldSR_X4_64_BSRGAN_PSNR.pth](https://github.com/mv-lab/swin2sr/releases/download/v0.0.1/Swin2SR_RealworldSR_X4_64_BSRGAN_PSNR.pth) v2 file named as `SwinIR_4x.v2.pth` in the `download` service, that is described in the comment:
* https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/2092#issue-1402318275

Best wishes,
Sergey.

**Comment by AbdBarho** [comment]
Hi @mrauhu,
if you want to use this model, download it and put it as `data/SwinIR/SwinIR_4x.v2.pth` and restart the UI, it should work out of the box.

I am unsure if this should be part of the download service since it is entirely optional, maybe we should have a separate `download-extra` for all of the additional stuff.

**Reply by mrauhu** [reply]
> if you want to use this model, download it and put it as `data/SwinIR/SwinIR_4x.v2.pth` and restart the UI, it should work out of the box.

Thank you.
